Source/Scene/PostProcessComposite.js

```diff
@@ -0,0 +1,237 @@
+        '../Core/createGuid',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject'
+    ], function(
+        Check,
+        createGuid,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject) {
+    'use strict';
+
+    /**
+     * A collection of {@link PostProcess}es or other post-process composites that execute together logically.
+     * <p>
+     * All processes are executed in the order of the array. The input texture changes based on the value of <code>executeInSeries</code>.
+     * If <code>executeInSeries</code> is <code>true</code>, the input to each post-process is the output texture rendered to by the scene or of the process that executed before it.
+     * If <code>executeInseries</code> is <code>false</code>, the input texture is the same for each post-process in the composite. The input texture is the texture rendered to by the scene
+     * or the output texture of the previous process or composite.
+     * </p>
+     *
+     * @alias PostProcessComposite
+     * @constructor
+     *
+     * @param {Object} options An object with the following properties:
+     * @param {Array} options.processes An array of {@link PostProcess}es or composites to be executed in order.
+     * @param {Boolean} [options.executeInSeries=true] Whether to execute each post-process where the input to one post-process is the output of the previous. Otherwise, the input to each contained post-process is the output of the post-process that executed before the composite.
+     * @param {String} [options.name=createGuid()] The unique name of this post-process for reference by other composites. If a name is not supplied, a GUID will be generated.
+     *
+     * @exception {DeveloperError} options.processes.length must be greater than 0.0.
+     *
+     * @see PostProcess
+     *
+     * @example
+     * // Example 1: seperable blur filter
+     * // The input to blurXDirection is the texture rendered to by the scene or the output of the previous process.
+     * // The input to blurYDirection is the texture rendered to by blurXDirection.
+     * scene.postProcessCollection.add(new Cesium.PostProcessComposite({
+     *     processes : [blurXDirection, blurYDirection],
+     * }));
+     *
+     * @example
+     * // Example 2: referencing the output of another post-process
+     * scene.postProcessCollection.add(new Cesium.PostProcessComposite({
+     *     executeInSeries : false,
+     *     processes : [
+     *         // The same as Example 1.
+     *         new Cesium.PostProcessComposite({
+     *             executeInSeries : true
+     *             processes : [blurXDirection, blurYDirection],
+     *             name : 'blur'
+     *         }),
+     *         // The input texture for this process is the same input texture to blurXDirection since executeInSeries is false
+     *         new Cesium.PostProcess({
```

Comment:
What is the point of this stage?  I get that the input is the previous external stage / rendered scene and that the previous stage's output is in the `blurTexture` uniform, but what is this example supposed to do with these?  Just a comment would help